pywhy_stats/independence/kci.py

```diff
@@ -61,12 +77,31 @@ def ind(
 
         kernel_X = func:`sklearn.metrics.pairwise.pairwise_kernels.polynomial`
 
+    If the kernel is a callable, it will have either one input for ``X``, or two inputs for ``X`` and
+    ``Y``. It is assumed that the kernel operates on the entire array to compute
+    the kernel array, that is, the callable performs a vectorized operation to compute the entire kernel matrix.
+
+    If one has an unvectorizable kernel function, then it is advised to use the callable with the
+    :func:`~sklearn.metrics.pairwise.pairwise_kernels` function, which will parallelize the kernel computation
+    across each row of the data.
```

Comment:
@bloebp this now describes the policy/behavior of the callable and string input for the `kernel_XYZ` parameter